api/db-cdr/generate-cdr/make-bq-data.sh

```diff
@@ -300,10 +300,23 @@ echo "Inserting concept_synonym"
 bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
 "INSERT INTO \`$WORKBENCH_PROJECT.$WORKBENCH_DATASET.concept_synonym\`
  (id, concept_id, concept_synonym_name)
-SELECT 0, c.concept_id, c.concept_synonym_name
+SELECT 0, c.concept_id, REPLACE(c.concept_synonym_name,"|","||") as concept_synonym_name
```

Comment:
The idea isn't to do the REPLACE here; you can remove this statement. It's to do the REPLACE in the string_agg above.

If you decide to do this, you don't need to put "\|" between synonyms; just | will suffice.